Documentation/operations/metrics.rst

```diff
@@ -530,6 +530,32 @@ Name                             Labels                                   Descri
 Options
 """""""
 
+This metric supports :ref:`Context Options<hubble_context_options>`.
+
+``flows-to-world``
+~~~~~~~~~~~~~~~~~~
+
+This metric counts flows that contain ``reserved:world`` label in the destination
+identity. By default, it counts dropped flows if and only if the drop reason is
+``Policy denied``. Set ``any-drop`` option to count all dropped flows.
```

Comment:
I've also been confused about what the metrics count and wrongly thought it only counted dropped flows. I would slightly reword the documentation to make it (hopefully) clearer:
```suggestion
This metric counts all flows containing the ``reserved:world`` label in their destination
identity. By default, dropped flows are counted if and only if the drop reason is
``Policy denied``. Set ``any-drop`` option to count all dropped flows.
```